src/public/bridge/windowbridge.h

```diff
@@ -17,8 +17,16 @@ int32_t WindowGetPosX(void);
 int32_t WindowGetPosY(void);
 bool WindowIsFullscreen(void);
 
+float ScreenGetAspectRatio(void);
+float LeftEdgeAlign(float v);
+int16_t LeftEdgeRectAlign(float v);
+float RightEdgeAlign(float v);
+int16_t RightEdgeRectAlign(float v);
+uint32_t WindowGetGameRenderWidth();
+uint32_t WindowGetGameRenderHeight();
```

Comment:
I know this is WIP still, but wanted to ask the question early.

Since these are coming from the WindowBridge and are `extern "C"` we might want to consider these all having a `Window` prefix to indicate the system they are coming from. Aligning with the recent LUS namespacing clean up efforts.

Reply:
But it's not really "WindowGetLeftEdge" it's more like "ScreenGetLeftEdge"

I don't mind adding Window, but just consider the name may be confusing.